app/components/NavButtons/index.js

```diff
@@ -1,21 +1,50 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import FontAwesome from 'react-fontawesome';
-
+import cx from 'classnames';
 import styles from './styles.scss';
 
-const NavButtons = props => {
-  return (
-    <div className={styles.nav_buttons}>
-      <a href='#' onClick={props.back} >
-        <FontAwesome name='chevron-left'/>
-      </a>
-      <a href='#' onClick={props.forward}>
-        <FontAwesome name='chevron-right'/>
-      </a>
-    </div>
-  );
-};
+class NavButtons extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.enableBackButton = this.enableBackButton.bind(this);
+    this.enableForwardButton = this.enableForwardButton.bind(this);
+  }
+
+  enableBackButton(currentHistoryIndex) {
+    if (currentHistoryIndex == 1) {
```

Comment:
Better to return this directly (and also use === instead of ==):

```javascript
return currentHistoryIndex === 1;
```

Similarly in the other function.